buildkite.yaml

```
@@ -8,6 +8,18 @@ steps:
       - "bundle"
       - "bundle exec rake"
     plugins:
+      - aws-assume-role-with-web-identity#v1.2.0:
+          role-arn: "${OIDC_ASSUME_ROLE_ARN:-arn:aws:iam::445615400570:role/pipeline-buildkite-test-collector-ruby-build}"
+          session-tags:
+          - organization_slug
+          - organization_id
+          - pipeline_slug
+          - build_branch
```

YAML is frustrating in its ambiguous indentation rules.
In this case, the array nested under `plugins` a few lines up has the hyphens indented past their container, which is my preferred style. Whereas this one has the hyphens a the same level as their container. My preference doesn't really matter, but we should probably be consistent within a single file.

```suggestion
            - organization_slug
            - organization_id
            - pipeline_slug
            - build_branch
```